README.md

```diff
@@ -166,7 +166,7 @@ including its children (total time).
 The `graph` command shows function call graph of given function.  In the above
 example, function graph of function 'main' looks like below:
 
-    $ uftrace graph  main
+    $ uftrace graph main
```

Comment:
I put an extra space here to indicate it separates uftrace command and user inputs.

Reply:
I understand. I undo commit.